README.md

```diff
@@ -147,3 +147,34 @@ The custom configuration options for the Celery workers are listed below:
   `{registry}/iib-build:{request_id}`.
 * `iib_log_level` - the Python log level for `iib.workers` logger. This defaults to `INFO`.
 * `iib_registry` - the container registry to push images to (e.g. `quay.io`).
+
+## Read the Docs Documentation
+
+### Build the Docs
+
+To build and serve the docs, run the following commands:
+```
+$ tox -e docs
+$ google-chrome .tox/docs_out/index.html
+```
+
+### Expanding the Docs
+
+To document a new Python module, find the `rst` file of the Python package that contains the module.
```

Comment:
Optional: `find the `rst` file of the Python package that contains the module` => `find the `rst` file of the corresponding Python package that contains the module`